File: 백준/Bronze/32342. 와우와 쿼리/와우와 쿼리.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int Q = Integer.parseInt(br.readLine());
        
        for (int a = 0; a < Q; a++) {
            String S = br.readLine();
            int tmp = 0;
            
            for (int i = 0; i <= S.length() - 3; i++) {
                if (S.substring(i, i + 3).equals("WOW")) {
                    tmp++;
                }
            }
            
            System.out.println(tmp);
        }
    }
}

File: 백준/Bronze/32342. 와우와 쿼리/와우와 쿼리.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    Q = int(input())
    for _ in range(Q):
        S = input()
        tmp = 0
        for i in range(len(S)-2):
            if S[i:i+3] == 'WOW':
               tmp += 1
        print(tmp)


if __name__ == '__main__':
    main()